src/inputs/button/buttonConstants.js

```diff
@@ -1,7 +1,140 @@
 import PropTypes from 'prop-types';
 import Utils from '../../utils/utils';
 
-export const buttonPropTypes = {
+export const VARIANTS = {
+    contained: 'contained',
+    outlined: 'outlined',
+    text: 'text',
+};
+
+export const buttonV2PropTypes = {
+    as: PropTypes.oneOf(['a', 'button']),
```

Comment:
NAB: Do we need a documentation comment on `as` prop?

Reply:
Sure, i can do this.